opennms-webapp/src/main/java/org/opennms/web/admin/notification/noticeWizard/NotificationWizardServlet.java

```diff
@@ -122,6 +122,7 @@ public void doPost(final HttpServletRequest request, final HttpServletResponse r
          */
         try {
             NotifdConfigFactory.init();
+            NotifdConfigFactory.getInstance().reload();
```

Comment:
is that possible to make the reload inside init ? It is kind of changing behaviour of the factory.

Reply:
Since init method is called before bean factory creation, configuration manager service is becoming null. So, moved that method to postconstruct.

Reply:
ok then.